content/ember/v3/ember-string-htmlsafe-ishtmlsafe.md

```diff
@@ -0,0 +1,34 @@
+---
+id: ember-string.htmlsafe-ishtmlsafe
+title: Deprecate importing htmlSafe and isHTMLSafe from @ember/string
+until: '4.0.0'
+since: '3.25'
```

Comment:
I was uncertain whether this should be `3.25` or `Upcoming features`.

Reply:
This is good